Dockerfile.sdk

```diff
@@ -205,6 +205,10 @@ RUN mkdir qa
 COPY qa/L0_sdk qa/L0_sdk
 COPY qa/L0_client_build_variants qa/L0_client_build_variants
 
+# Create a directory for all the python client tests to enable unit testing 
+RUN mkdir -p qa/python_client_unit_tests/ 
+COPY --from=sdk_build /workspace/client/src/python/library/tests/* qa/python_client_unit_tests/
```

Comment:
Nit: To make this consistent with the previous COPY commands and prevent COPY from being called multiple times on each file in the directory, you may want to update this line to:
`COPY --from=sdk_build /workspace/client/src/python/library/tests qa/python_client_unit_tests`